### ISSUE TITLE
[Feature request] Option + Discard Changes to discard without confirmation

### ISSUE THREAD

**Issue by iconara** [ForkIssues/Tracker#2264]
I often use the Discard Changes buttons on hunks in the diff view, and I would love to be able to speed up my workflow by for example holding down option to discard a hunk without any confirmation dialog.

**Comment by DanPristupov** [comment]
No, this will not happen in the future. I don't want users to speed up the discard. The discard function should not be easily accessible.

**Comment by iconara** [comment]
Not even hunks that are just a few lines, and with a modifier key pressed?
